doc/configcommands.dsv

```diff
@@ -8,7 +8,7 @@ bind-key||<key> <operation> [<dialog>]||n/a||Bind key <key> to <operation>. This
 bookmark-autopilot||[yes/no]||no||If set to `yes`, the configured bookmark command is executed without any further input asked from user, unless the url or the title cannot be found/guessed.||bookmark-autopilot yes
 bookmark-cmd||<command>||""||If set, then <command> will be used as bookmarking plugin. See the documentation on bookmarking for further information.||bookmark-cmd "~/bin/delicious-bookmark.sh"
 bookmark-interactive||[yes/no]||no||If set to `yes`, then the configured bookmark command is an interactive program.||bookmark-interactive yes
-browser||<command>||%BROWSER, otherwise lynx||Set the browser command to use when opening an article in the browser. If BROWSER environment variable is set, it will be used as the default browser, otherwise lynx will be used. Any occurrences of `%u` in <command> will be replaced by a URL in single quotes.||browser "w3m %u"
+browser||<command>||%BROWSER, otherwise lynx||Set the browser command to use when opening an article in the browser. If BROWSER environment variable is set, it will be used as the default browser, otherwise lynx will be used. Any occurrences of `%u` and `%U` in <command> will be replaced by the article's URL or the feed's URL, respectively, enclosed in single quotes.||browser "w3m %u"
```

Comment:
This is a good start, but it doesn't cover the case when `%u` is used and user tries to open a feed. [In that case](https://github.com/newsboat/newsboat/pull/437/files#diff-e953778e3f493f71c3eaec051e841374R230), `%u` is either the "alternative URL" of the feed, or its real URL if the alternative is not known. I find it quite tricky to put into words succinctly; feel free to have a try at it :) Or just roll back this change and I'll try myself, later.

Reply:
I think we should add a paragraph (and/or table) explaining the details somewhere else and just provide a brief example here. I'll see what I can come up with.

Reply:
This table is a reference, there can be only one example (in the last column). A separate table idea isn't that bad, though I'm not sure where to put it. Just after this "commands" table perhaps?

Reply:
Yeah, the example provided is fine, I should've been clearer. I was thinking about something along the lines of:

```
[...] Any occurences of '%u' and '%U' in <command> will be replaced according to [insert link to  the browser formatting options]. If <command> contains neither '%u' nor '%U' newsboat will behave as though ' %u' was appended to <command>, implicitly.
````

Then, put a table at the end of the docs along with the other formatting options tables.

Reply:
Sounds good, I'd merge that!

Reply:
Ping. If you finished these changes, can you push them? No rush, just want to make sure you haven't abandoned this yet.